licenses/Sourcetrail

```diff
@@ -0,0 +1,132 @@
+Sourcetrail
```

Comment:
It is the contents of the EULA.txt file from the dist tarball, and I didn't modify it in anyway, I thought it might be nice to have it in it's original form, so that it would be easier to detect if the license text is updated. Is that ok?

Reply:
Yes, that's even disable given that lawyers could argue adding line breaks changes something

Reply:
Oh, one more thing. Please add it to `@EULA` group in `profiles/license_groups` (another commit).

Reply:
I have added an extra commit with this (hope I got it right) :)